homeassistant/components/sensor/nsw_fuel_station.py

```diff
@@ -0,0 +1,120 @@
+"""
+Component to display the current fuel prices at a NSW fuel station.
+
+For more details about this component, please refer to the documentation at
+https://home-assistant.io/components/nsw_fuel_station/
+"""
+import datetime
+from typing import Optional
+import voluptuous as vol
```

Comment:
Please keep the import ordered -> https://developers.home-assistant.io/docs/en/development_guidelines.html